setup.py

```diff
@@ -12,21 +12,22 @@
 
 setup(
     name='django-pyfs',
-    version='2.0',
+    version='2.1',
     description='Django pyfilesystem integration',
     author='Piotr Mitros',
     author_email='pmitros@edx.org',
     packages=['djpyfs'],
     license="Apache 2.0",
     url="https://github.com/edx/django-pyfs",
     long_description=ld,
+    long_description_content_type='text/x-rst',
```

Comment:
I'm pretty sure this should be `text/markdown` instead; `README.md` apparently doesn't contain anything that isn't also valid reStructuredText so it passed the twine check, but I suspect it would lose some of the block formatting on PyPI if uploaded like this.